_posts/2016/2016-07-07-wiremock.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: "How not to produce 'Distributed Balls of Mud'"
+date: "2016-07-07 12:00:00"
+image: /blog-header/balls-of-mud.jpg
```

Comment:
I adapted the image size to our needs. Please use the one attached:
![balls-of-mud](https://cloud.githubusercontent.com/assets/10460958/16609648/00dff472-4356-11e6-9380-60e6fb9f550f.jpg)